### PR TITLE
Make a proposal's repository-folder title available as variable for sablon templates.

### LINKED ISSUE #1560: Ordnungsposition-Namen in Protokoll darstellen
Gemäss Kundenwunsch will man den Ordnungspositions-Titel auch im Protokoll:

https://feedback.onegovgever.ch/t/feedback-zur-neuen-aktuell-aufgestalteten-version-januar-2016/214

Umsetzung: Bei einem Antrag kann eine Sprache ausgewählt werden, der Titel wird dann nur in dieser Sprache gespeichert. Falls im System nur eine Sprache existiert passiert die Auswahl automatisch.

### PR DESCRIPTION
**Pull request by deiferni**
This PR makes the repo-folder title of a proposal's dossier available for sablon templates. The language of the proposal has to be chosen upon creation, the repo-folder title is then stored on a propsal in that language.

Closes #1560.

This PR also contains a fix for the test setup and adds better handling for content with a translated title.

### CODE REVIEW

**Comment by deiferni** [comment]
@lukasgraf plz have a look

File: opengever/testing/builders/translated.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from opengever.base.behaviors.translated_title import TranslatedTitle
+
+
+class TranslatedTitleBuilderMixin(object):
+    """Set translated title attribtues 'correctly' during tests.
```
Comment (lukasgraf) [inline opengever/testing/builders/translated.py]: typo: `attribtues`

File: opengever/meeting/profiles/default/metadata.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <metadata>
-    <version>4631</version>
+    <version>4633</version>
```
Comment (lukasgraf) [inline opengever/meeting/profiles/default/metadata.xml]: Is this gap just for the purpose of Kevin-Friendlyness, or is this an oversight? :wink:

File: opengever/meeting/upgrades/to4632.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    def migrate(self):
+        self.add_columns()
+        self.migrate_data()
+        self.make_columns_non_nullable()
+
+    def add_columns(self):
+        self.op.add_column(
+            'proposals',
+            Column('repository_folder_title', Text, nullable=True))
```
Comment (lukasgraf) [inline opengever/meeting/upgrades/to4632.py]: ``Text` should be `UnicodeCoercingText`, no?`

File: opengever/meeting/tests/test_proposal.py [hunk 4]
```diff
@@ -159,6 +164,51 @@ def test_proposal_can_be_edited_in_browser(self, browser):
         self.assertEqual(u'Lorem ips\xfcm', model.proposed_action)
 
     @browsing
+    def test_proposal_language_field_with_multiple_languages(self, browser):
+        """Create and update proposal with multiple languages.
+
+        Test that form displays language select and updates/sets
```
Comment (lukasgraf) [inline opengever/meeting/tests/test_proposal.py]: -> `language selection widget`?

**Comment by lukasgraf** [comment]
Otherwise: Looks good to me :+1:

File: opengever/meeting/upgrades/to4632.py (hunk 3 above)
Comment (deiferni) [inline opengever/meeting/upgrades/to4632.py]: As discussed, this is just for the upgrade where `Text` is a safe choice. `UnicodeCoercingText` is a type decorator used for runtime conversion.

File: opengever/meeting/profiles/default/metadata.xml (hunk 2 above)
Comment (deiferni) [inline opengever/meeting/profiles/default/metadata.xml]: there are two upgrades :wink:

**Comment by deiferni** [comment]
@lukasgraf updated & tested on MySQL

**Comment by lukasgraf** [comment]
:+1: :tada:

**Pull request merged**